app/components/frame-viewer.spec.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import assert from 'assert';
+import { mount } from 'enzyme';
+import FrameViewer from './frame-viewer';
+import FrameAnnotator from '../classifier/frame-annotator';
+
+const subject = {
+  locations: [{ image: 'www.testimage.com' }]
+};
+
+const videoSubject = {
+  locations: [{ video: 'www.testvideo.com' }]
+};
+
+const clickSimulate = {
```

Comment:
This variable name doesn't really match what it is doing. Neither sinon nor the built in enzyme simulate method is being called here.